### DISCUSSION TITLE
Which Chinese companies can work remotely?

### DISCUSSION THREAD

**Discussion by martinadamsdev** [remote-camps/how-to-work-remotely#1] [Announcements]
If you know of a Chinese company where you can work remotely, please leave a comment.

- company name
- official website
- Is anything else you know

We sincerely thank you for your contribution. 💯

**Comment by yuetong3yu** [comment]
Company name: **Tower**
Official website: [Tower](https://tower.im/)

👋 A quite cool small company. Since my company's choice, I used their product a lot, and by the chance knowing the team from [Tower 团队 48 个月远程实践](https://tower.im/help/articles/73).
